src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -28,25 +28,29 @@ const createErrorMessage = (fetchResult: FetchResultType) => {
   }, '')
 }
 
-export default function * fetchResourceCache (city: string, language: string, fetchMap: FetchMapType): Saga<ResourceCacheStateType> {
+export default function * fetchResourceCache (city: string, language: string, fetchMap: FetchMapType, database: MemoryDatabase): Saga<void> {
+  yield call(database.readResourceCache)
+
+  if (isEmpty(fetchMap)) {
+    return
```

Comment:
I think we should always fire an action here. In this case a success action should be fired.

Reply:
Done.